### PR TITLE
add redux

### PR DESCRIPTION
**Pull request by Abel-desu**
Understand the concept of state management.
Use store, actions, and reducers in React.

### CODE REVIEW

File: src/Redux/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>
+    </Provider>
+  </React.StrictMode>,
+);
```
Comment (TedLivist) [inline src/Redux/index.js]: - [ ] From the file structure shared with you this file should be in the `src` directory, not `src/Redux`. Kindly fix that.

File: src/Redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+const categoriesReducer = (state = [], action) => {
+  if (action.type === CHECK_STATUS) {
+    return ['Under Construction'];
```
Comment (TedLivist) [inline src/Redux/categories/categories.js]: - [ ] In keeping with the requirement this should be a string, not an array. Kindly refactor accordingly.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Abel-desu,

Good job so far!
✔️ Implementation of action creators
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Abel-desu, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

# Optional: Kindly update your readme with a project title according to the project and also instead of Author1, you can write your own name.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**